helmfile/README.md

```diff
@@ -0,0 +1,105 @@
+# Graph Protocol indexer helmfile
+Graph protocol indexer could be installed in two different modes:
+* Network mode - participating in network, process requests and get commision
+* Standalone mode - private installation that would be used, managed only by owner.
```

Comment:
maybe worth rephrasing a bit

Reply:
Is it better now?

Reply:
`node doesn't join the graph network, processes only your own subgraph and requests.`